clients/zbctl/cmd/complete.go

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// completeCmd represents the complete command
+var completeCmd = &cobra.Command{
```

Comment:
Why we need this file?

Reply:
zbctl follow the structure `zbctl ACTION RESOURCE`. So this is the sub command for the complete action